### ISSUE TITLE
Unable to mark text

### ISSUE THREAD

**Issue by cbjerg** [viktorhajer/simple-pdf-viewer#16]
I am unable to select text in the pdf. I need it for a screen reader that will read the selected text out loud. Is this possible somehow?

**Comment by viktorhajer** [comment]
I should be working as you can see in the demo app: 
https://viktorhajer.github.io/simple-pdf-viewer/

There is a flag to disable the text layer but it is false by default and it can't be changed from outside. Are you sure your PDF document has text layer? (maybe there is no text layer, e.g. scanned book...)

**Comment by cbjerg** [comment]
Sorry - my bad.
I had implemented a gesture recognizer for swiping between pages on iPad, which ruined the highlighting feature.
